lang_GENERIC/analyze/Dataflow_constness.ml

```diff
@@ -35,6 +35,19 @@ module DataflowX = Dataflow.Make (struct
       Display_IL.short_string_of_node_kind n.F.n
   end)
 
+(*****************************************************************************)
+(* Error management *)
+(*****************************************************************************)
+
+let warning tok s =
```

Comment:
Actually we probably want to hook in the error reporting of semgrep, so use the Error_code.error function,
but it's good enough for now.